java/hillshade-renderer/README.md

```diff
@@ -1,38 +1,32 @@
-# Hillshade Renderer
-Use a hillshade renderer on a raster.
+# Hillshade renderer
 
-![Hillshade Renderer App](hillshade-renderer.png)
+Apply a hillshade renderer to a raster.
+
+![Image of hillshade renderer](hillshade-renderer.png)
+
+## Use case
+
+An environmental agency may track coastal erosion by comparing images of an area taken over a longer period of time with hillshade renderers applied.
 
 ## How to use the sample
-Tap on the wrench button in the Action Bar to change the settings for the `HillshadeRenderer`. The sample allows you to change the `Altitude`, `Azimuth` and `Slope type`. You can tap on the `Render` button to update the raster.
+
+Choose and adjust the settings to update the hillshade renderer on the raster layer. The sample allows you to change the Altitude, Azimuth, and Slope Type.
```

Comment:
Again, maybe it's worth adding that the settings are available from the wrench icon in the top right.

Reply:
ditto above.